bin/thoth

```diff
@@ -50,26 +48,26 @@ var thoth = {
 	logs: 'logs',
 	log: 'log',
 	verbose: true,
-	asset: 'thoth.tar.gz',
+	// FIXME
+	asset: '/thoth/public/thoth-sunos-1.3.0.tar.gz',
 	db: {
 		host: 'localhost',
 		port: 28015,
 		db: '',
 		authKey: 'bagnoogle',
 		table: 'dumps'
 	},
-	indexed: {
+	sysprops: {
```

Comment:
What is the difference here, and will it cause a problem with anything else expecting `indexed`?

Reply:
That was dead code. 'sysprops' is a new feature so you can 'thoth set' stuff outside of properties {}